rootfs/api/models/app.py

```diff
@@ -129,7 +132,65 @@ def delete(self, *args, **kwargs):
         self._clean_app_logs()
         return super(App, self).delete(*args, **kwargs)
 
-    def restart(self, **kwargs):
+    def restart(self, **kwargs):  # noqa
+        """
+        Restart found pods by deleting them (RC will recreate).
+        Wait until they are all drained away and RC has gotten to a good state
+        """
+        try:
+            # Fetch the initial set of pods to work from
+            pods = self.list_pods(**kwargs)
+            desired = len(pods)
+        except KubeException:
+            # Nothing was found
+            return []
+
+        try:
+            for pod in pods:
```

Comment:
oh wait this is the delete, not the "new things are coming up" loop :) Hmm yeah I should probably more that to threading even if I hate the idea of it and very much so want to kill all Threading in here... Maybe in the future I will move it to `asyncio` but I will have a look at `Threading` for now :( :( :( :(